include/ginkgo/core/solver/cb_gmres.hpp

```diff
@@ -144,6 +144,13 @@ class CbGmres : public EnableLinOp<CbGmres<ValueType>>,
      */
     size_type get_krylov_dim() const { return krylov_dim_; }
 
+    /**
+     * Sets the krylov dimension
```

Comment:
nit: (also in GMRES)
```suggestion
     * Sets the Krylov dimension
```